src/phy/mod.rs

```diff
@@ -161,6 +161,8 @@ pub struct ChecksumCapabilities {
     pub tcpv4: Checksum,
     #[cfg(feature = "proto-ipv4")]
     pub icmpv4: Checksum,
+    #[cfg(feature = "proto-ipv4")]
+    pub igmp: Checksum,
```

Comment:
Is there any hardware that computes IGMP checksums? If not I'd prefer to not have this.

Reply:
Removed!